onadata/libs/tests/utils/test_analytics.py

```diff
@@ -31,7 +32,7 @@ def test_get_user_id(self):
         self.assertTrue(len(user2.email) > 0)
         self.assertEqual(get_user_id(user2), user2.email)
 
-    @override_settings(SEGMENT_WRITE_KEY='123', HOSTNAME='test-server')
+    @override_settings(SEGMENT_WRITE_KEY='123')
```

Comment:
Could we add back the `HOSTNAME` setting? Would be nice to see how the Analytics payload looks with it.